toolkit/tools/imagegen/attendedinstaller/views/installerview/installerview.go

```diff
@@ -147,6 +147,11 @@ func (iv *InstallerView) Primitive() tview.Primitive {
 
 // OnShow gets called when the view is shown to the user
 func (iv *InstallerView) OnShow() {
+	err := speakuputils.StartSpeakup()
```

Comment:
Why do we start speakup both here and in the init script?

Reply:
See above comment.